source/blueprints/banking.rst

```diff
@@ -0,0 +1,546 @@
+
+   Experiment with different models, features, and hyperparameter selection settings to see how they affect classifier performance. Maintain a held-out validation set to evaluate your trained NLP models and analyze misclassified test instances. Then, use observations from the error analysis to inform your machine learning experimentation. See the :doc:`User Guide <../userguide/nlp>` for examples and discussion.
+
+
+9. Parser Configuration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The relationships between entities in the banking assistant queries are simple ones. For example, in the annotated query ``can you pay my {minimum|credit_amount} bill``, the ``credit amount`` entity is self-sufficient, in that it is not described by any other entity.
+
+If you extended the app to support queries with more complex entity relationships, it would be necessary to specify *entity groups* and configure the parser accordingly. For example, in the query ``can you pay my {minimum|credit_amount} bill for my {amex|credit_card}``, we would need to relate the ``credit_amount`` entity to the ``credit_card`` entity, because one entity describes the other. The related entities would form an entity group. For more about entity groups and parser configurations, see the :doc:`Language Parser <../userguide/parser>` chapter of the User Guide.
+
+Since we do not have entity groups in the banking assistant, we do not need a parser configuration.
+
+
+10. Using the Question Answerer
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :doc:`Question Answerer <../userguide/kb>` component in MindMeld is mainly used within dialogue state handlers for retrieving information from the knowledge base. Since the banking assistant has no knowledge base, no question answerer is not needed.
+
+
+11. Testing and Deployment
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once all the individual pieces (NLP, Question Answererer, Dialogue State Handlers) have been trained, configured, or implemented, use the :class:`Conversation` class in MindMeld to perform an end-to-end test of your conversational app.
+
+For instance:
+
+.. code:: python
+
+   from mindmeld.components.dialogue import Conversation
+   conv = Conversation(nlp=nlp, app_path='banking_assistant')
+   conv.say("pay off my credit card bill for my remaining balance please")
+
+.. code-block:: console
+
+   ['Ok we have scheduled your credit card payment for your balance of $5000']
+
+The :meth:`say()` method packages the input text in a user request object and passes it to the MindMeld Application Manager to simulate a user interacting with the application. The method then outputs the textual part of the response sent by the app's Dialogue Manager. In the above example, we requested to pay off our credit debt, in a single query. The app responded, as expected, with the amount that will be paid off.
+
+You can also try out multi-turn dialogues(transfer balances):
+
+.. code:: python
+
+   >>> conv = Conversation(nlp=nlp, app_path='banking_assistant')
```

Comment:
You can also pass the username in the context

Reply:
Yes, would be good to show an example with the username passed into the context since that's one of the new things in this blueprint.